src/main/java/org/exschool/pocketworld/model/Time.java

```diff
@@ -0,0 +1,31 @@
+package org.exschool.pocketworld.model;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+
+@Entity
+public class Time {
+	
+	private int level;
```

Comment:
should contains such fields:
int level (building`s level)
BuildingType(enum that represents type of building)
int time (amount of minutes for building construction)